dev/com.ibm.ws.cdi.api_fat/publish/servers/cdi12InjectInjectionPointParamServer/jvm.options

```diff
@@ -0,0 +1,2 @@
+# Set a system property.
+-Dcom.ibm.ws.webbeans.crossEjbModuleBindings=true
```

Comment:
What??

Reply:
this is for cdi 1.0. It must a c+p error

Reply:
It is a c+p error, I copied the entire server folder.